lib/mayaUsd/ufe/PulledObjectHierarchyHandler.cpp

```diff
@@ -69,7 +69,10 @@ Ufe::SceneItem::Ptr PulledObjectHierarchyHandler::createItem(const Ufe::Path& pa
     return _mayaHierarchyHandler->createItem(path);
 }
 
-Ufe::Hierarchy::ChildFilter PulledObjectHierarchyHandler::childFilter() const { return {}; }
+Ufe::Hierarchy::ChildFilter PulledObjectHierarchyHandler::childFilter() const
+{
+    return _mayaHierarchyHandler->childFilter();
```

Comment:
In the chain of responsibility we cannot assume that Maya handlers have no child filters: the proxy shape hierarchy handler adds a child filter flag for inactive prims, since its children are USD prims.

Reply:
Looks good.